libqtile/backend/wayland/core.py

```diff
@@ -1286,26 +1287,19 @@ def unhide_cursor(self) -> None:
     @expose_command()
     def get_inputs(self) -> dict[str, list[dict[str, str]]]:
         """Get information on all input devices."""
-        info = {}
-        device_lists: dict[str, list[inputs._Device]] = {
-            "type:keyboard": self.keyboards,  # type: ignore
-            "type:pointer": self._pointers,  # type: ignore
-        }
-
-        for type_key, devices in device_lists.items():
-            type_info = []
-
-            for dev in devices:
-                type_info.append(
-                    dict(
-                        name=dev.wlr_device.name,
-                        identifier=dev.get_info()[1],
-                    )
-                )
-
-            info[type_key] = type_info
+        info: DefaultDict[str, list[dict]] = defaultdict(lambda: [])
+        devices: list[inputs._Device] = self.keyboards + self._pointers  # type: ignore
+
+        for dev in devices:
+            dev_info = dev.get_info()
+            type_key = dev_info[0]
+            type_info = dict(
+                name=dev.wlr_device.name,
+                identifier=dev_info[1],
+            )
+            info[type_key] += [type_info]
```

Comment:
We can avoid creating the throwaway list by just appending:
```suggestion
            type_key, identifier = dev.get_info()
            type_info = dict(
                name=dev.wlr_device.name,
                identifier=identifier,
            )
            info[type_key].append(type_info)
```

Reply:
And that makes a lot of sense